sdks/python/apache_beam/io/textio.py

```diff
@@ -147,7 +147,7 @@ def display_data(self):
 
   def read_records(self, file_name, range_tracker):
     start_offset = range_tracker.start_position()
-    read_buffer = _TextSource.ReadBuffer('', 0)
+    read_buffer = _TextSource.ReadBuffer(b'', 0)
```

Comment:
You're right. 
I haven't focused too much on the `io` files yet, since this package still needs to be ported. However, since it is related to this change, I will already add it.